gateway/handler_success.go

```diff
@@ -170,17 +171,11 @@ func (s *SuccessHandler) RecordHit(r *http.Request, timing Latency, code int, re
 			// mw_redis_cache instead? is there a reason not
 			// to include that in the analytics?
 			if responseCopy != nil {
-				contents, err := ioutil.ReadAll(responseCopy.Body)
```

Comment:
This code change causes `TestRedisCacheMiddleware_WithCompressedResponse` to fail.